### ISSUE TITLE
Update "Do disabled elements (<input>, <button>, etc) need to pass minimum contrast ratio?"

### ISSUE THREAD

**Issue by Jefferydo** [cenw-wscoe/WCAG-in-Plain-Language#53]
The answer goes off topic in the first paragraph and in the "More Information" section.

The "More Information" section says all HTML5 elements pass contrast out of the box, but doesn't mention that's not the case for the placeholder attribute. The "Exception" addresses the placeholder, but on the wrong topic (failing 3.3.2 if it's the only label). 

Course page on the topic: [Styling the placeholder](https://bati-itao.github.io/learning/esdc-self-paced-web-accessibility-course/module6/form-instructions.html#styling-the-placeholder)

Best to stick to contrast and delete the unrelated advice. Address those topics in their own questions.

Incidentally, does WCAG discuss a disabled submit button, specifically? I have no doubt it's bad practice, but can't find mention of it.
